python/src/DistFunc_doc.i.in

```diff
@@ -583,185 +671,1303 @@ Examples
+             \left(\delta(x - \gamma)
+             \sqrt{\frac{2}{\nu + x^2}}\right) ^ j,
+             \quad x \in \Rset
+
+With :math:`\Gamma` denotes Euler's Gamma function
+:meth:`~openturns.SpecFunc.Gamma`.
 
-Returns
--------
-realizations : :class:`~openturns.Point` or :class:`~openturns.Sample`
-    The realizations of the uniform disctribution.
 
 Examples
 --------
 >>> import openturns as ot
 >>> ot.RandomGenerator.SetSeed(0)
->>> a = [1, 5, 0]
->>> b = [2, 3, -1]
->>> c = [-1, 2, 2]
->>> d = [1, 2, 1]
->>> r = ot.DistFunc.rUniformTetrahedron(a, b, c, d)
->>> r = ot.DistFunc.rUniformTetrahedron(a, b, c, d, 10)"
+>>> r = ot.DistFunc.rNonCentralStudent(2.5, 0.5)
+>>> r = ot.DistFunc.rNonCentralStudent(2.5, 0.5, 10)"
 
 // ---------------------------------------------------------------------
 
-%feature("docstring") OT::DistFunc::rUniformSimplex
-"Realization of a uniform distribution over a simplex.
+%feature("docstring") OT::DistFunc::pNonCentralStudent
+"Cumulative distribution function  of a NonCentralStudent distribution.
 
 Parameters
 ----------
-vertices : 2-d sequence of float, a list of points in :math:`\Rset^n`
-    The vertices of the simplex.
-size : int
-    The number of realizations to generate.
+nu : float, :math:`\nu > 0`,
+    The :math:`\nu` parameter.
+delta : float, :math:`\delta > 0`,
+    The :math:`\nu` parameter.
+x : float, 
+    Location.
+tail : bool, default=False
+    Tail flag.
+    
+    If True, the complementary cdf is computed.
+
+Returns
+-------
+p: float,
+    The cdf or the complementary cdf at x.
 
 Notes
 -----
-This method samples the barycentric coordinates, allowing to work in any dimension.
+The position parameter  :math:`\gamma` is equal to zero.
+
+The probability density function is defined as:
+
+.. math::
+
+    f_X(x) = \frac{exp \left(-\delta^2 / 2 \right)}
+                  {\sqrt{\nu\pi} \Gamma \left(\frac{\nu}{2} \right)}
+             \left(\frac{\nu}{\nu + x^2}\right) ^ {\frac{\nu + 1}{2}}
+             \sum_{j=0}^{\infty}
+             \frac{\Gamma \left(\frac{\nu + j + 1}{2}\right)}{\Gamma(j + 1)}
+             \left(\delta(x - \gamma)
+             \sqrt{\frac{2}{\nu + x^2}}\right) ^ j,
+             \quad x \in \Rset
+
+With :math:`\Gamma` denotes Euler's Gamma function
+:meth:`~openturns.SpecFunc.Gamma`.
+
+We use the Viktor Witkovsky's algorithm described in [witkovsky2003]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> cdf = ot.DistFunc.pNonCentralStudent(2.5, 3.5, 9.5)
+>>> cdf = ot.DistFunc.pNonCentralStudent(2.5, 3.5, 9.5, True)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::qStudent
+"Quantile of a scalar Student distribution.
+
+
+Parameters
+----------
+nu : float, :math:`\nu > 0`,
+    The :math:`\nu` parameter.
+p : float, in :math:`[0,1]`,
+    The probability.
+tail : bool, default=False,
+    Tail flag.
+
+    If True, the quantile associated to :math:`(1-p)` is computed.
```

Comment:
```suggestion
    If True, the quantile associated to :math:`1 - p` is computed.
```

Reply:
You missed this one apparently.

Reply:
I think it is better to maintain the ().